WISE-Community/WISE
# Verify tests for admin controllers

Test that the tests in portal.presentation.web.controllers.admin work:
* FindProjectRunsControllerTest (1 test)
* UserInfoControllerTest (1 test)
* ViewAllUsersControllerTest (3 tests)

1. Open file in IntelliJ, right click anywhere in file -> run test. Check that all tests in each file pass.
2. run "mvn test" in command line. All tests should pass.
